CMakeLists.txt

```diff
@@ -19,9 +19,6 @@ include(cmake/base.cmake)
 include(cmake/cython/cython.cmake)
 include(cmake/msvc-specific.cmake)
 
-# Disable -Werror on Unix for now.
-set(CXX_DISABLE_WERROR True)
```

Comment:
So I create one option per target? One for RBDyn, one for the parsers and one for the tests/benchmarks?

Reply:
No. As long as it is enabled for RBDyn it will propagate for the targets that link with RBDyn (only in the current build, hence the `$<BUILD_INTERFACE>` expression)